src/contracts/Protocol/QuestionAPI.sol

```diff
@@ -46,7 +46,7 @@ contract QuestionAPI is Ownable, NFTLocked {
         _costController.payForCreateQuestion(_msgSender());
 
         // Mint a new question
-        uint256 questionId = _question.safeMint(_msgSender(), uri);
+        uint256 questionId = _question.createQuestion(_msgSender(), uri);
```

Comment:
there's an argument against naming every method the same as it becomes more difficult in debugging? something like mintQuestion IDK but we above you have tokenID but here it's question we shoudl remain consistent shoudl like hodl

Reply:
yeah you're right, we shoudl - fuck it we're still minting I'll rename